src/main/java/com/shika/pages/ACourseCategoryPage.java

```diff
@@ -47,10 +48,24 @@ public ACourseCategoryPage(WebDriver driver) {
 	@FindBy(how = How.ID, using = "toolbar-trash")
 
 	private WebElement trash;
+	
+	@FindBy(how = How.ID, using = "toolbar-batch")
+	
+	private WebElement batch;
+	
+    @FindBy(how = How.XPATH, using = "//*[@id=\"batch-access\"]")
+	
+	private WebElement setAccessLevel;
 
 	@FindBy(how = How.ID, using = "filter_search")
 
 	private WebElement textboxSearch;
+	
+	@FindBy(how = How.ID, using = "cb0")
+	public WebElement check1;

```

Comment:
Don't use  variable names as "check1", "check2", etc. Use specific names, if its for publish and unpublish, use "btnPublish" and "btnUnPublish" accordingly.